db/user_test.go

```diff
@@ -14,6 +15,83 @@ type SyncNewUsersTestCase struct {
     sendEmails bool
 }
 
+type ResolveUsersTestCase struct {
+    input []string
+    expectedOutput []string
+    course *model.Course
+    actualOutput []string
+    err error
+}
+
+func (this *DBTests) DBTestResolveUsers(test *testing.T) {
+    defer ResetForTesting();
+
+    for i, testCase := range getResolveUsersTestCases() {
+        ResetForTesting();
+        if (i == 4) {
```

Comment:
Nope, never do this.
The index for a test case should only ever be used for reference (when outputting an error).
A test case should contain all the information it needs to test itself.